File: src/me/tcklpl/naturaldisaster/shop/ShopCommand.java
package me.tcklpl.naturaldisaster.shop;

import me.tcklpl.naturaldisaster.GameStatus;
import me.tcklpl.naturaldisaster.NaturalDisaster;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class ShopCommand implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command cmd, String alias, String[] args) {
        if (cmd.getName().equalsIgnoreCase("shop") || alias.equalsIgnoreCase("loja")) {
            if (args.length != 0) return false;
            if (sender instanceof Player) {
                Player p = (Player) sender;
                if (NaturalDisaster.getGameManager().getCurrentStatus() == GameStatus.IN_LOBBY) {
                    Shop s = new Shop(p);
                    s.show();
                } else {
                    p.sendMessage(ChatColor.RED + "Você só pode abrir a loja no lobby, caso o game já tenha acabado aguarde alguns segundos.");
                }
                return true;
            }
        }
        return false;
    }
}

File: src/main/java/me/tcklpl/naturaldisaster/shop/ShopCommand.kt
package me.tcklpl.naturaldisaster.shop

import me.tcklpl.naturaldisaster.GameStatus
import me.tcklpl.naturaldisaster.NaturalDisaster
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

object ShopCommand : CommandExecutor {

    override fun onCommand(sender: CommandSender, cmd: Command, alias: String, args: Array<String>): Boolean {
        if (args.isNotEmpty()) return false
        if (sender !is Player) return false

        if (NaturalDisaster.instance.gameManager.currentStatus == GameStatus.IN_LOBBY) {
            val s = Shop(sender)
            s.show()
        } else {
            sender.sendMessage("${ChatColor.RED}Você só pode abrir a loja no lobby, caso o game já tenha acabado aguarde alguns segundos.")
        }
        return true
    }
}
